### PR TITLE
Manipulator interactor

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

File: Sources/Interaction/Manipulators/CameraManipulator/api.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+## Introduction
+
+vtkCameraManipulator is a superclass foractions inside an interactor style
```
Comment (aronhelser) [inline Sources/Interaction/Manipulators/CameraManipulator/api.md]: typo -> for actions

File: Sources/Interaction/Manipulators/CameraManipulator/api.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+and associated with a single button. An example might be rubber-band
+bounding-box zoom. This abstraction allows a camera manipulator to be
+assigned to any button.  This super class might become a subclass of
+vtkInteractorObserver in the future.
+
+## Settings
+
+These settings determine which button and modifiers the manipulator responds
+to. Button can be either 1 (left), 2 (middle), and 3 right.
```
Comment (aronhelser) [inline Sources/Interaction/Manipulators/CameraManipulator/api.md]: add parens (right) for consistency

File: Sources/Interaction/Manipulators/CameraManipulator/api.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Set/Get the rotation factor, default is 1.
+
+### manipulatorName
+
+Set/Get the manipulator name.  Default is empty string.
+
+## Methods
+
+The following methods are abstract and must be implmented by subclasses
```
Comment (aronhelser) [inline Sources/Interaction/Manipulators/CameraManipulator/api.md]: typo: implemented

**Comment by jourdain** [comment]
@aronhelser, I created that MR before the code was ready.
I did it to help me follow the code when @scottwittenburg push to his remote branch.

**Comment by aronhelser** [comment]
OK, just a few things I noticed scanning through...

**Comment by jourdain** [comment]
LGTM

**Pull request merged**